ui/src/domain/errors/api-error.ts

```diff
@@ -0,0 +1,8 @@
+export class ApiError extends Error {
+    
+    private readonly errorStatus: number;
+    constructor(errorStatus: number, message: string) {
+        super(message)
```

Comment:
```suggestion
        super(message);
```
I'm surprised this wasn't picked up by the linter.

Reply:
Done